Library/Formula/pam_u2f.rb

```diff
@@ -0,0 +1,38 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
```

Comment:
Given the project-name is `pam-u2f` it'd be good if the formula was named `pam-u2f.rb` (rather than underscore)